### ISSUE TITLE
All page content should be contained by landmarks

### ISSUE THREAD

**Issue by MikeTschudi** [Esri/solutions-components#1629]
#solutions-deploy-app,#calcite-tab-title-3a5a9943-21f3-1e41-90fc-ed5d6155cb88

### Violation Details:

**Rule:** region
**Rule Name:** Ensure all page content is contained by landmarks
**Description:**  All page content should be contained by landmarks
**Standard:**  Deque Best Practices
**WCAG Criteria:**  Deque Best Practices
**Violations:**  2
**Impact:**  Moderate

 https://dequeuniversity.com/rules/axe/4.10/region?application=axeAPI

- **Element Location:** #solutions-deploy-app,#calcite-tab-title-3a5a9943-21f3-1e41-90fc-ed5d6155cb88
- **Element Source:** "< calcite-tab-title aria-selected="true" id="calcite-tab-title-3a5a9943-21f3-1e41-90fc-ed5d6155cb88" role="tab" tabindex="0" layout="inline" calcite-hydrated="" selected="" aria-controls="calcite-tab-title-54c80630-c9e2-5828-363f-716c0f38cc22" >< !--?lit$461391299$-- >All Solutions< /calcite-tab-title >"
- **Suggested Fix:** "Fix any of the following:  Some page content is not contained by landmarks"
- **Screenshot Reference:** https://desktopqa/psregression/Accessibility/-1735383313_region.png
